### PR TITLE
Add file list function

### PR DESCRIPTION
**Pull request by chel4**

### CODE REVIEW

File: deadline_fighters/desktop-client/index.html [hunk 1]
```diff
@@ -1,171 +1,176 @@
...
+                //				});
+                //
+
+            });
+
+
+            //list object
+            bucket.listObjects({}, function(err, data) {
+              if (err) console.log(err, err.stack); // an error occurred
+              $('#objectList').empty();   //empty the list
```
Comment (sranjani-s) [inline deadline_fighters/desktop-client/index.html]: Shouldn't the rest of it be under 'else'?

File: deadline_fighters/desktop-client/index.html [hunk 2]
```diff
@@ -1,171 +1,176 @@
...
+              	var params4 = {
+              		Bucket: 'deadlinefighters',
+              		CopySource: '/deadlinefighters/Test1！.zip',
+              		Key: '2！.zip',
+                };
+                re_name.copyObject(params4, function (err, data) {
+                     if (err) console.log(err, err.stack);
+                     else {
+                  	        console.log(data);
+                            re_name.deleteObject(params3, function (err, data) {
```
Comment (sranjani-s) [inline deadline_fighters/desktop-client/index.html]: Fix text alignment please.

File: deadline_fighters/desktop-client/index.html [hunk 3]
```diff
@@ -1,171 +1,176 @@
...
+              }
+              downloadFile(FilePath1, bucket1, key1);
+        		});
+
+
+
+            //delete
+            var bucket2 = 'deadlinefighters';
+            var key2 = '！.txt';
+            var dele_te = new AWS.S3();
```
Comment (sranjani-s) [inline deadline_fighters/desktop-client/index.html]: The variable name can be a little better? dele_te is confusing from del.

File: deadline_fighters/desktop-client/index.html [hunk 4]
```diff
@@ -1,171 +1,176 @@
...
+      <ul  id="objectList" >     </ul>
+
+
+
+
+
+      <script type="text/javascript">
+        		/// <reference types="aws-sdk" />
+        		/// Load the SDK for JavaScript
+               // var AWS = require('aws-sdk');
```
Comment (sranjani-s) [inline deadline_fighters/desktop-client/index.html]: Please remove unnecessary comments.

File: deadline_fighters/desktop-client/index.html (hunk 2 above)
Comment (chel4) [reply deadline_fighters/desktop-client/index.html]: I will fix the alignment at final.

**Pull request merged**